core/iwasm/libraries/libc-builtin/libc_builtin_wrapper.c

```diff
@@ -345,9 +345,21 @@ sprintf_out(int c, struct str_context *ctx)
     return c;
 }
 
-#ifdef BH_PLATFORM_OPENRTOS
-PRIVILEGED_DATA static char print_buf[128] = { 0 };
-PRIVILEGED_DATA static int print_buf_size = 0;
+#ifndef BUILTIN_LIBC_BUFFERED_PRINTF
+#define BUILTIN_LIBC_BUFFERED_PRINTF 0
+#endif
+
+#if BUILTIN_LIBC_BUFFERED_PRINTF != 0
+#ifndef BUILTIN_LIBC_BUFFERED_PRINT_SIZE
+#define BUILTIN_LIBC_BUFFERED_PRINT_SIZE 128
+#endif
+#ifndef BUILTIN_LIBC_BUFFERED_PRINT_PREFIX
+#define BUILTIN_LIBC_BUFFERED_PRINT_PREFIX
+#endif
+
+BH_BUILTIN_LIBC_BUFFERED_PREFIX static char
+    print_buf[BH_BUILTIN_LIBC_BUFFERED_SIZE] = { 0 };
+BH_BUILTIN_LIBC_BUFFERED_PREFIX static int print_buf_size = 0;
```

Comment:
Should remove BUILTIN_LIBC_BUFFERED_PRINT_PREFIX and BUILTIN_LIBC_BUFFERED_PRINT_SIZE